### PR TITLE
introducting Plans

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
To be merged onces plans are live
requires https://github.com/Drakkar-Software/OctoBot-Blog/pull/23

### CODE REVIEW

File: 2023-02-cloud-octobot-plans.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Introducing Cloud OctoBot plans
+subtitle: Get your the perfect Cloud OctoBot
```
Comment (Herklos) [inline 2023-02-cloud-octobot-plans.md:3]: 👍

File: 2023-02-cloud-octobot-plans.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+In the OctoBot team, we want everyone to be able to use OctoBot and enjoy great trading strategies.  
+
+Our goal is that whether you want to get a cheap and pre-configured strategy or infinite customization to create your very own strategy, you should be able to do it using OctoBot.
+
+That's why we created 4 different plans that are designed to fit every user. 
+
+# The plans
+
+## Starter
+> Comming soon
```
Comment (Herklos) [inline 2023-02-cloud-octobot-plans.md]: I think we will make it available in this update. I will still have the web interface.

File: 2023-02-cloud-octobot-plans.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+The Starter plan is perfect for investors who look for simplicity. Automate a trading strategy out of the box on a few trading pairs of your choice. 
+
+The Starter plan is using the simplest version of OctoBot and is also the cheapest one.
+
+## Essential
+The Essential plan is designed for traders who want to try different trading strategies, customize them and trade using many trading pairs.
```
Comment (Herklos) [inline 2023-02-cloud-octobot-plans.md:27]: 👍

File: 2023-02-cloud-octobot-plans.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+Our goal is that whether you want to get a cheap and pre-configured strategy or infinite customization to create your very own strategy, you should be able to do it using OctoBot.
+
+That's why we created 4 different plans that are designed to fit every user. 
+
+# The plans
+
+## Starter
+> Comming soon
+
+The Starter plan is perfect for investors who look for simplicity. Automate a trading strategy out of the box on a few trading pairs of your choice.
```
Comment (Herklos) [inline 2023-02-cloud-octobot-plans.md:22]: 👍

File: 2023-02-cloud-octobot-plans.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+The Pro plan is made for advanced traders who are using advanced strategy design tools, large history backtesting and want to get the very best out of their trading strategies. 
+
+Using the Pro plan, you will have access to an improved version of OctoBot, which includes the exclusive Strategy Editor interface.
+
+## Self hosting
+
+The Self hosting plan allows you to run your OctoBot directly from home, on the device of your choice. It is free and relies on you to install, run and maintain your OctoBot.
+
+If you prefer to run OctoBot at home or on your own server, the Self hosting plan is made for you. Please note that some features of the Essential or Pro plan might not be available in self hosted OctoBots.
```
Comment (Herklos) [inline 2023-02-cloud-octobot-plans.md:42]: 👍

File: 2023-02-cloud-octobot-plans.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+# How to try Cloud OctoBot plans ?
+
+Select a strategy on [OctoBot Cloud](https://www.octobot.cloud) and enjoy a free trial period of the cloud OctoBot plans.
+
+> Additionally, to celebrate the launch of Cloud OctoBots, we are running a time limited early bird offer which heavily discounts all the plans. This is a rare chance to get your Cloud OctoBot at a very cheap price.
```
Comment (Herklos) [inline 2023-02-cloud-octobot-plans.md:48]: 👍

File: 2023-02-cloud-octobot-plans.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+You will find all the plans details on the [bots section of OctoBot Cloud](https://octobot.cloud/bots).
+
+# Why limiting traded pairs and exchanges on Cloud OctoBots ?
+
+When hosting a program on a cloud, costs come mainly from RAM and processing power. 
+
+- Thanks to the many optimizations included in OctoBot, processing power is not an issue. 
+
+- However, the more enabled exchanges and trading pairs, the more RAM is necessary to run an OctoBot. This is especially true when running a backtest on a large history. This is the reason we have to limit the maximum amount of simultaneous exchanges and traded pairs depending on the selected plan. In order to offer cheaper hosting prices, each plan relies on a specific hosting setup in accordance with these limits.
```
Comment (Herklos) [inline 2023-02-cloud-octobot-plans.md:60]: 👍

Comment (Herklos) [reply 2023-02-cloud-octobot-plans.md:60]: Good idea to explain it

File: 2023-02-cloud-octobot-plans.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+We are currently working on increasing those limits to offer our customers the best experience possible.
```
Comment (Herklos) [inline 2023-02-cloud-octobot-plans.md:62]: 👍

File: 2023-02-cloud-octobot-plans.md (hunk 2 above)
Comment (GuillaumeDSM) [reply 2023-02-cloud-octobot-plans.md]: Updated

**Review by Herklos — APPROVED** [review]

**Pull request merged**